testsuite/openshift/objects/ingress.py

```diff
@@ -69,16 +71,17 @@ def create_service_ingress(cls, openshift: 'OpenShiftClient', name, service_name
         return cls.create_instance(openshift, name, rules=[rule])
 
     @property
-    def rules(self):
```

Comment:
as it already has been implemented I would keep the rules property present. I think that it will find its use soon.